CONTRIBUTING.md

```diff
@@ -418,6 +418,9 @@ Be sure to post in the PR conversation that you have made the requested changes.
 
 ### Other resources
 
+-   [Challenge Style Guide to create a new freeCodeCamp
```

Comment:
Or just "Style Guide for freeCodeCamp Challenges", which I think implies new and existing challenges :smile: